src/frontend/lib/logger.ts

```diff
@@ -0,0 +1,22 @@
+import Bugsnag from "@bugsnag/react-native";
+// import semverPrerelease from "semver/functions/prerelease";
+// import { version } from "../../../package.json";
+
+// const prereleaseComponents = semverPrerelease(version);
+
+Bugsnag.start({
+  /**
+   * TODO(andrew): release stage cannot be set from JS layer and must be done at native layer
```

Comment:
highlighting this as this PR still needs to address this issue (unless we're okay with not accounting for prerelease release stages?)

Reply:
yes, let me know if you need help with this.

Reply:
I probably will honestly. Don't have the greatest familiarity around native android configurations 😅

Reply:
Made an attempt to solve this but the confidence level is somewhere betwee low and middling 😅 . See https://github.com/digidem/mapeo-mobile/pull/867/commits/a873b72d3ac9a202044996e89bb88192e7f18d5d